### ISSUE TITLE
option to fail a read if not all series are found

### ISSUE THREAD

**Issue by HomoCodens** [mbannert/timeseriesdb#231]
In a couple of projects now I've seen code fail after a `db_ts_read(con, some_keys)` because not all series in `some_keys` are present in the resulting list.

Rather than have each piece of code that relies on all series being read having to do a check we could add a "fail_if_missing" kind of option to `db_ts_read` that stops with a meaningful error if not all the requested keys can be read (because of access rights, release dates, missing altogether etc.).

**Comment by HomoCodens** [comment]
Or alternatively at least always give a warning so the error is easier to track down.

**Comment by omuelle** [comment]
Yes, I had similar issues. At least a warning would be nice. The reason should be given, e.g. insufficient access rights.
